contracts/actions/reflexer/ReflexerNativeUniV2SaviourDeposit.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../ActionBase.sol";
+import "./helpers/ReflexerHelper.sol";
+import "../../utils/TokenUtils.sol";
+
+/// @title Deposit lpToken in the contract in order to provide cover for a specific SAFE managed by the SAFE Manager
+contract ReflexerNativeUniV2SaviourDeposit is ActionBase, ReflexerHelper {
+    using TokenUtils for address;
+
+    address public constant UNDERLYING_NATIVE_TOKEN_ADDRESS = 0x8aE720a71622e824F576b4A8C03031066548A3B1;
```

Comment:
Weird name, this is the uniV2 lp token for rai/weth maybe something shorter more desciprive, UNIV2_RAI_WETH_ADDRESS